README.md

```diff
@@ -41,6 +41,7 @@ jobs:
     steps:
       - uses: nearform/optic-release-automation-action@main # you can use a tag instead of main
         with:
+          all-secrets: ${{ toJson(secrets) }}
```

Comment:
I'm not 100% sure that this is a viable option. This means that somebody using the action will have to trust sending all the secrets to the action. Before deciding that we don't have other options, have we considered alternatives?

Reply:
I suspected this might be an issue even though the action is public. I can try to filter the secrets using regex(there's an action for that) and only send the NPM/Optic token related secrets to the action.

Reply:
Yes, let's see if there's anything else here which could help https://docs.github.com/en/actions/learn-github-actions/expressions